spec/features/profiles_spec.rb

```diff
@@ -19,4 +19,22 @@
       expect( page ).to have_content(@comment.body)
     end
   end
+  
+  describe "signed in" do
+    before do
+      login_as(authenticated_user, scope: :user)
+    end
+    
+    it "shows profile" do
+      visit user_path(@user)
+      expect(current_path).to eq(user_path(@user))
+      expect( page ).to have_content(@user.name)
+      expect( page ).to have_content(@post.title)
+      expect( page ).to have_content(@comment.body)
+    end
+    
+    after do
+      Warden.test_reset!
```

Comment:
because of the `config.after :each` in `rails_helper.rb`, this should already be run after each test. what happens when you remove this?